core.spec.md

```diff
@@ -0,0 +1,501 @@
+
+###! feature: String!
+
+A feature URL specifying the directive and associated schema elements. When viewed, the URL SHOULD provide the content of the appropriate version of the specification in some human-readable form. In short, a human reader should be able to click the link and go to the docs for the version in use. There are specific requirements on the format of the URL, but it is not required that the *content* be machine-readable in any particular way.
+
+Feature URLs contain information about the spec's [prefix](#sec-Prefixing) and [version](#sec-Versioning).
+
+Feature URLs serve two main purposes:
+  - Directing human readers to documentation about the feature
+  - Providing tools with information about the specs in use, along with enough information to select and invoke an implementation
+  
+Feature URLs SHOULD be [RFC 3986 URLs](https://tools.ietf.org/html/rfc3986). When viewed, the URL SHOULD provide the specification of the selected version of the feature in some human-readable form; a human reader should be able to click the link and go to the correct version of the docs.
+
+Although they are not prohibited from doing so, it's assumed that processors will not load the content of feature URLs. Published specifications are not required to be machine-readable, and [this spec](.) places no requirements on the structure or syntax of the content to be found there.
+
+There are, however, requirements on the structure of the URL itself:
+
+```html diagram -- Basic anatomy of a feature URL
+<code class=anatomy>  
+  <span class=pink style='--depth: 2'>https://spec.example.com/a/b/c/<span>exampleFeature<aside>name</aside></span><aside>identity</aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+</code>
+```
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) MUST contain the feature's name and a [version tag](#sec-Versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the feature's *identity*. For the above example,
+<dl>
+  <dt>`identity: "https://spec.example.com/a/b/c/exampleFeature"`</dt>
+  <dd>A global identifier for the feature. Processors can treat this as an opaque string identifying the feature (but not the version of the feature) for purposes of selecting an appropriate implementation.</dd>
+  <dt>`name: "exampleFeature"`</dt>
+  <dd>The feature's name, for purposes of [prefixing](#sec-Prefixing) schema elements it defines or extends.</dd>
+  <dt>`version: "v1.0"`</dt>
+  <dd>The tag for the [version](#sec-Versioning) of the feature used to author the document. Processors MUST select an implementation of the feature which can [satisfy](#sec-Satisfaction) the specified version.</dd>
+</dl>
+
+The version tag MUST be a valid {VersionTag}.
```

Comment:
SUBSTANTIVE: Should we not specify that the feature's name must be valid (a GraphQL name without the namespace separator)?

Reply:
Fixed.